app/controllers/plans_controller.rb

```diff
@@ -38,9 +38,13 @@ def new
   def create
     @plan = Plan.new
     authorize @plan
-
+    
     @plan.principal_investigator = current_user.surname.blank? ? nil : "#{current_user.firstname} #{current_user.surname}"
-    @plan.data_contact = current_user.email
+    @plan.principal_investigator_email = current_user.email
+    
+    orcid = current_user.identifier_for(IdentifierScheme.find_by(name: 'orcid'))
```

Comment:
I agree.  I was jusg suggesting that we save the `IdentifierScheme.find_by(name: 'orcid')` within the model for identifier_schemes instead of running a query each time we want to select it.  just like Perms, the content of the record for the orcid identifier scheme will remain constant while the tool is used